pageserver/src/page_service.rs

```diff
@@ -642,7 +642,7 @@ impl std::fmt::Display for BatchedPageStreamError {
 struct BatchedGetPageRequest {
     req: PagestreamGetPageRequest,
     timer: SmgrOpTimer,
-    effective_request_lsn: Lsn,
+    lsns: LsnRange,
```

Comment:
nit: why not `lsn_range` instead instead of `lsns`?

Reply:
Changed